Makefile.machine.include

```diff
@@ -39,6 +39,9 @@ BSG_MACHINE_DRAM_BANK_SIZE_WORDS      = 134217728
 
 BSG_MACHINE_DATA_WIDTH                = 32
 
+# BSG Machine Name is used to build non-conflicting crt.o files
+BSG_MACHINE_NAME                      = Y$(BSG_MACHINE_GLOBAL_Y)X$(BSG_MACHINE_GLOBAL_X)_NONBLK_DDR4_F1MODEL
```

Comment:
I think it should be somewhat similar format as mem cfg names.
e.g.) y12_x8_vcache_non_blocking_ddr4_f1model

Reply:
I think this is good feedback. I'd be happy to do this too. 

I deferred to shorter names, but I'd be happy with your suggestion too.

Reply:
I'm going to roll back the name addition. @vb000 is right, recompilation is not needed for crt.o